doc/howto/gh-actions-badges.md

```diff
@@ -0,0 +1,84 @@
+# Use GitHub Actions To Place A MyBinder.org Badge On Pull Requests
```

Comment:
```suggestion
# Use GitHub Actions to place a MyBinder.org badge on pull requests
```

The other titles seem to capitalize like this suggestion.

![image](https://user-images.githubusercontent.com/3837114/87814944-fd4a2100-c864-11ea-8f69-ee7edd43c7a3.png)

Reply:
This was marked as resolved without any comment, perhaps I did it by mistake, so I unresolved this.

Reply:
Ohh I think I accidentally comittted over this, my bad